source/deployment-paradigms/multi-region.txt

```diff
@@ -62,6 +62,133 @@ multiple geographies, ensures the highest level of availability with a single
 provider. If your application requires the very highest level of availability 
 and lowest latency, consider a :ref:`arch-center-paradigms-multi-cloud`.
 
+Multi-Region Deployment Use Cases
+---------------------------------
+
+The following use cases highlight a high-criticality application ("Tier 1") 
+and a less-critical application, but one that still warrants a 
+multi-region deployment ("Tier 2").
+
+.. _arch-center-multi-region-tier-1:
+
+Tier 1 Applications
+~~~~~~~~~~~~~~~~~~~
+
+Tier 1 applications require both high availability and low-latency. 
+To achieve this, we recommend an architecture that consists of a minimum of 5 
+nodes spread over at least 3 regions. This provides high availability in at least 
+2 regions. If a node goes down, the traffic can be served from the same 
+region to ensure the same latency on the primary region. 
+
+.. figure:: /includes/images/multi-region-tier-1.png
+   :figwidth: 750px
+   :alt: An image showing three types of multi-region deployments
+
+Notes and Considerations
+````````````````````````
+
+- Your application should also be deployed in each region to further reduce  
+  latency.
+
+- You should use Privatelink for connecting to the cluster, and VPC peering 
+  between the your VPCs. VPC peering ensures that if a network connection 
+  is broken or {+service+} in that region goes down, the application tier can 
+  still route to the primary node, first over the vpc peering, and then over the 
+  private endpoint. 
+- This architecture has the highest cost due to network traffic between regions 
+  and having a 5 or more data-bearing nodes. 
+- This architecture provides the highest resiliency. There are no interruptions 
+  to operations, and your application can sustain a full regional failure with 
+  no interruption and manual intervention required. 
+
+.. _arch-center-multi-region-tier-2:
+
+Tier 2 Applications
+~~~~~~~~~~~~~~~~~~~
+
+If you have an application that can experience limited downtime or maintenance 
+windows without significant revenue impact, you can create an architecture that 
+provides that level of availability by using 5 nodes in 2 regions. You have 3 
+electable nodes spread across the 2 regions, and also have 2 read-only nodes in 
+the secondary region. The electable nodes are in the **majority region**, while 
+the region with fewer electable nodes is the **minority region**.
+
+.. figure:: /includes/images/multi-region-tier-2.png
+   :figwidth: 750px
+   :alt: An image showing three types of multi-region deployments
+
+Notes and Considerations
+````````````````````````
+This is a good option for:
+
+- Customers with only 2 approved regions.
+- Meeting governance compliance about data storage.
+- Applications where read operations need high uptime, but write operations can 
+  experience some outage.
```

Comment:
I think this information should appear first, so users know early whether or not this applies to them.